app/Models/Loggable.php

```diff
@@ -55,11 +55,19 @@ public function logCheckout($note, $target /* What are we checking out to? */)
         if ($log->target_type == Location::class) {
             $log->location_id = $target->id;
         } elseif ($log->target_type == Asset::class) {
-            $log->location_id = $target->rtd_location_id;
+            $log->location_id = $target->location_id;
         } else {
             $log->location_id = $target->location_id;
         }
 
+        if ($log->item_type == Asset::class) {
+
+            if ($asset = Asset::find($log->item_id)) {
+                \Log::debug('Increment the checkout count for asset: '.$log->item_id);
+                $asset->increment('checkout_counter', 1);
+            }
+        }
+
```

Comment:
Is this better to do in Asset::checkOut() ? It would avoid the asset conditional and it's more related to checking out than logging a checkout, I think.

Reply:
Sure, I can move it. Was having a hard time backtracking everything that happens on checkin/checkout. (I hate those methods and really want to refactor.)